xbmc/pvr/PVRPlaybackState.cpp

```diff
@@ -363,6 +363,13 @@ void CPVRPlaybackState::StartPlayback(CFileItem* item,
         client->StreamClosed();
 
       client->GetChannelStreamProperties(item->GetPVRChannelInfoTag(), source, props);
+
+      if (props.LivePlaybackAsEPG())
+      {
+        const std::shared_ptr<CPVREpgInfoTag> epgTag = item->GetPVRChannelInfoTag()->GetEPGNow();
+        if (epgTag)
+          item = new CFileItem(epgTag);
```

Comment:
This introduces a memory leak. You need to delete `item` before assigning it a new value. item is a raw pointer.

Reply:
Fixed